tests/chainer_tests/training_tests/triggers_tests/test_interval_trigger.py

```diff
@@ -1,80 +1,136 @@
 from __future__ import division
 
+import mock
+import numpy as np
+import random
+import tempfile
 import unittest
 
+from chainer import serializers
 from chainer import testing
 from chainer import training
 
 
-class DummyUpdater(training.Updater):
+def get_trainer_with_mock_updater(iter_per_epoch):
```

Comment:
I noticed we can avoid using `IntervalTrigger` in `get_trainer_with_mock_updater` by setting `stop_trigger=None`. To solve later issue, I added `iter_per_epoch` option to `get_trainer_with_mock_updater` #2913.